pkg/controller/kieapp/defaults/defaults_test.go

```diff
@@ -6035,3 +6035,308 @@ func createKafkaJbpmObject(dateFormat string, tasksTopics string, casesTopics st
+}
+
+func checkAcAllowMethod(t *testing.T, cors *api.CORSFiltersObject) {
+	assert.Equal(t, cors.AllowMethodsName, "Access-Control-Allow-Methods")
+	assert.Equal(t, cors.AllowMethodsValue, "GET, POST, OPTIONS, PUT")
+}
+
+func checkAcAllowHeaders(t *testing.T, cors *api.CORSFiltersObject) {
+	assert.Equal(t, cors.AllowHeadersName, "Access-Control-Allow-Headers")
+	assert.Equal(t, cors.AllowHeadersValue, "Accept, Authorization, Content-Type, X-Requested-With")
+}
+
+func checkAcAllowCredentials(t *testing.T, cors *api.CORSFiltersObject) {
+	assert.Equal(t, cors.AllowCredentialsName, "Access-Control-Allow-Credentials")
+	assert.True(t, *cors.AllowCredentialsValue)
+}
+
+func checkAcMaxAge(t *testing.T, cors *api.CORSFiltersObject) {
+	assert.Equal(t, cors.MaxAgeName, "Access-Control-Max-Age")
+	assert.Equal(t, cors.MaxAgeValue, Pint32(1))
+}
+
+func checkConsoleCORSAssertions(t *testing.T, cr *api.KieApp, env api.Environment) {
+	corsEnabled := isCORSEnabled(env.Console.DeploymentConfigs[0].Spec.Template.Spec.Containers[0].Env)
+	assert.True(t, corsEnabled)
+	assert.NotNil(t, cr.Spec.Objects.Console)
+	cors := cr.Status.Applied.Objects.Console.Cors
+	assert.NotNil(t, cors)
+	checkCors(t, cors)
+}
+
+func checkConsoleCustomCORSAssertions(t *testing.T, cr *api.KieApp, env api.Environment) {
+	corsEnabled := isCORSEnabled(env.Console.DeploymentConfigs[0].Spec.Template.Spec.Containers[0].Env)
+	assert.True(t, corsEnabled)
+	assert.NotNil(t, cr.Spec.Objects.Console)
+	cors := cr.Status.Applied.Objects.Console.Cors
+	assert.NotNil(t, cors)
+	checkCustomCors(t, cors)
+}
+
+func checkKieServerCORSAssertions(t *testing.T, cr *api.KieApp, env api.Environment) {
+	corsEnabled := isCORSEnabled(env.Servers[0].DeploymentConfigs[0].Spec.Template.Spec.Containers[0].Env)
+	assert.True(t, corsEnabled)
+	assert.Len(t, cr.Spec.Objects.Servers, 1)
+}
+
+func checkDashbuilderCORSAssertions(t *testing.T, cr *api.KieApp, env api.Environment) {
+	corsEnabled := isCORSEnabled(env.Dashbuilder.DeploymentConfigs[0].Spec.Template.Spec.Containers[0].Env)
+	assert.True(t, corsEnabled)
+	assert.NotNil(t, cr.Spec.Objects.Dashbuilder)
+}
+
+func isCORSEnabled(envs []corev1.EnvVar) bool {
+	corsEnabled := false
+	for _, env := range envs {
+		if strings.HasPrefix(env.Name, "AC_ALLOW") || strings.HasPrefix(env.Name, "FILTERS") {
+			if env.Name == "FILTERS" && len(env.Value) > 9 { //AC_ALLOW_
+				corsEnabled = true
+			} else if env.Name != "FILTERS" && env.Value != "<nil>" && len(env.Value) > 1 {
+				corsEnabled = true
+			}
+		}
+	}
+	return corsEnabled
+}
+
+func TestEnvironmentWithoutCORS(t *testing.T) {
+	cr := &api.KieApp{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "test-no-cors",
+		},
+		Spec: api.KieAppSpec{
+			Environment: api.RhdmTrial,
+			Objects: api.KieAppObjects{
+				Console: &api.ConsoleObject{},
+				Servers: []api.KieServerSet{},
+			},
+		},
+	}
+
+	env, err := GetEnvironment(cr, test.MockService())
```

Comment:
instead add a entire test for it, remove it and add the lines like assert.Nil(t, cr.Status.Applied.Objects.Console.Cors) on a existing test,

Reply:
ok